assets/styles/viper.css

```diff
@@ -1,5 +1,7 @@
 body {
   background-color: white;
+  width: 100%;
```

Comment:
ok, will temporarily accept. I see that when the page loads, the size is OK and in a fraction of a second it goes to this size. There are no JS errors in console.